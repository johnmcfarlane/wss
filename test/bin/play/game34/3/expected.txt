 51  3,1  | LEVIER
 48  7,3  - CEBIL
 42  7,3  - CELEB
 42  7,3  - CLEVE
 42  7,3  - CLIVE
 39  3,1  | BELIER
 39  3,1  | EVILER
 39  3,1  | LIEVER
 39  3,6  | RELIVE
 39  3,6  | REVEIL
 39  3,6  | REVILE
 39  3,1  | VEILER
 36  3,1  | LIEFER
 36  3,6  | REFILE
 36  3,6  | RELIEF
 32  4,4  - BELOVE
 30  7,3  - CEILE
 30  8,4  | FI
 25  2,8  | BEEF
 24  2,1  | EVIL
 24  3,3  | FEBRILE
 24  2,1  | LEIF
 24  2,1  | VEIL
 24  2,1  | VILE
 23  3,5  | BRIEVE
 23  2,2  | ELF
 22  2,1  | BILE
 22  2,1  | FEIL
 22  2,1  | FELE
 22  2,1  | FILE
 22  2,2  | LIF
 22  4,4  | VENIE
 22  3,4  | VERBILE
 22  2,2  | VIL
 21  4,1  | BEEFIN
 21  6,4  - BOVE
 21  2,1  | FBI
 21  2,8  | LEVE
 21  4,2  | LEVEN
 21  2,8  | LEVI
 21  4,2  | LEVIN
 21  4,2  | LIVEN
 21  4,6  | NEVEL
 20  6,6  | ALIVE
 20  6,6  | AVILE
 20  3,4  | BERVIE
 20  3,5  | BRIEF
 20  1,4  - ELEVE
 20  3,3  | FIBRE
 20  2,2  | FIL
 20  6,4  - LOVEE
 20  7,4  - OVILE
 20  2,1  | VEL
 20  3,3  - VELIC
 20  2,1  | VIL
 19  2,8  | BEL
 19  5,5  | BILEVE
 19  4,3  | EVEN
 19  5,4  - FLOB
 19  2,8  | LEEF
 19  2,8  | LEIF
 19  2,8  | LEV
 19  6,4  - LOVE
 19  4,6  | NEVE
 19  4,6  | NEVI
 19  4,4  - VELO
 19  2,2  | VIE
 19  4,4  | VINE
 19  1,8  - aLIVE
 18  6,6  | ABELE
 18  6,6  | ALBEE
 18  6,6  | ALIFE
 18  2,1  | BEL
 18  5,3  | BELIVE
 18  3,5  | BREVE
 18  3,5  | BREVI
 18  4,2  | ELFIN
 18  3,5  | ERVIL
 18  2,4  | FEEL
 18  2,4  | FIEL
 18  2,1  | FIL
 18  4,2  | LEBEN
 18  2,4  | LEEF
 18  1,3  - LEVEL
 18  2,4  | LIEF
 18  4,2  | LIFEN
 18  3,3  | LIVRE
 18  4,6  | NEBEL
 18  4,6  | NIFLE
 18  7,4  - OBV
 18  4,4  | VIN
 17  2,8  | BEE
 17  5,3  | BEFILE
 17  5,3  | BELIEF
 17  5,3  | BEVIL
 17  3,3  - ELFIC
 17  3,4  | FERV
 17  2,2  | FIE
 17  6,4  - FOB
 17  7,4  - OBELI
 17  2,1  | VEI
 16  4,4  | BENE
 16  4,4  | BENI
 16  4,4  - BILO
 16  4,4  | BINE
 16  4,4  - BLEO
 16  3,5  | BREV
 16  7,3  - CIVE
 16  4,3  | EBEN
 16  4,5  | ENIF
 16  4,4  - FILO
 16  4,4  | FINE
 16  1,3  - LIBEL
 16  3,3  | LIBRE
 16  6,4  - LOBE
 16  6,4  - LOBI
 16  4,6  | NIFE
 16  7,4  - OBLI
 16  3,4  | VERB
 16  5,3  - VICE
 16  1,8  - aLIFE
 16  1,9  - nEVEL
 15  3,2  | BEFIRE
 15  4,4  | BEN
 15  5,2  | BEVEIL
 15  4,4  | BIN
 15  3,3  | BLERE
 15  7,3  - CIV
 15  2,9  | ELB
 15  2,9  | ELF
 15  2,9  | EVE
 15  4,2  | FELINE
 15  4,4  | FEN
 15  4,4  | FIN
 15  6,4  - FOIBLE
 15  3,5  | FRIB
 15  7,5  - GIBEL
 15  7,5  - GIVE
 15  6,4  - LOB
 15  6,4  - LOF
 15  4,6  | NEB
 15  4,6  | NEF
 15  4,6  | NIB
 15  7,4  - OBL
 15  3,6  | REIVE
 15  5,3  - VIC
 15  3,5  | VRIL
 15  1,8  - aLEF
 15  1,8  - aLIF
 14  3,2  | BEVER
 14  5,3  - BICE
 14  3,5  - EEL
 14  2,6  | ELB
 14  4,5  | ENLIVE
 14  4,5  | ENVEIL
 14  3,4  | FERLIE
 14  3,2  | FEVER
 14  5,3  - FICE
 14  3,2  | FIVER
 14  2,2  | ILE
 14  1,3  - LBF
 14  1,3  - LEVE
 14  1,3  - LEVI
 14  3,3  | LIBR
 14  1,3  - LIVE
 14  7,4  - OBE
 14  7,4  - OBI
 14  7,4  - OLIVE
 14  3,3  | VEER
 14  3,3  | VIER
 14  5,3  | VILI
 14  5,3  | VLEI
 14  1,9  - nEBEL
 13  5,3  - BEC
 13  5,3  | BELIE
 13  2,2  | BI
 13  3,2  | BIFER
 13  4,3  | BLIN
 13  3,5  | BRL
 13  7,3  - CLIV
 13  4,5  - EF
 13  4,7  - EF
 13  3,2  | ELVER
 13  4,5  | ENFILE
 13  4,5  | ENLIEF
 13  4,5  | ENLIFE
 13  1,4  - EVIL
 13  5,3  - FEC
 13  2,4  | FEE
 13  2,2  | FI
 13  3,2  | FIBER
 13  2,4  | FIE
 13  4,3  - FLIC
 13  6,4  - FOLIE
 13  7,5  - GIBE
 13  7,5  - GLB
 13  2,3  | IF
 13  4,5  - IF
 13  3,4  | IFREE
 13  4,5  | INFL
 13  1,3  - LEV
 13  4,4  - LEVO
 13  2,2  | LIE
 13  1,3  - LIV
 13  7,4  - OF
 13  3,6  | RFB
 13  5,4  - VIOL
 13  6,4  - VOLE
 13  1,8  - aVILE
 13  1,9  - nIFLE
 12  6,3  - BCF
 12  3,3  | BEER
 12  4,3  - BEICE
 12  3,3  | BIER
 12  5,4  - BIOL
 12  3,4  | BIRLE
 12  3,2  | BLIER
 12  6,4  - BOLE
 12  3,5  | BREE
 12  3,5  | BREI
 12  3,5  | BRIE
 12  7,3  - CLEF
 12  2,1  | EEL
 12  4,5  | ENVIE
 12  5,7  - FE
 12  3,3  | FEER
 12  5,7  - FI
 12  5,3  | FILI
 12  3,2  | FLEER
 12  3,2  | FLIER
 12  3,4  | FLRIE
 12  6,4  - FOLE
 12  3,5  | FREE
 12  7,5  - GEB
 12  7,5  - GIB
 12  7,5  - GIF
 12  7,5  - GLEBE
 12  3,2  | LEVER
 12  3,2  | LEVIR
 12  1,3  - LIEVE
 12  1,3  - LIFE
 12  4,4  - LIFO
 12  3,2  | LIVER
 12  1,3  - LLB
 12  4,6  | NIEVE
 12  3,6  | REEF
 12  3,6  | REIF
 12  3,6  | REVEL
 12  3,6  | RIFLE
 12  3,6  | RIVEL
 12  3,2  | VILER
 12  3,4  | VIRL
 12  6,4  - VOILE
 12  6,4  - VOL
 12  1,8  - aBELE
 12  1,9  - nEVE
 12  1,9  - nEVI
 11  3,4  | BIRL
 11  6,4  - BOL
 11  2,9  | EF
 11  1,4  - ELVE
 11  3,5  | ERF
 11  3,2  | FILER
 11  5,5  | FIVE
 11  6,4  - FOL
 11  7,5  - GLIB
 11  2,8  | LEE
 11  2,1  | LEI
 11  2,8  | LEI
 11  5,3  | LEVI
 11  1,3  - LIB
 11  3,2  | LIBER
 11  5,5  | LIEVE
 11  1,3  - LIF
 11  3,2  | LIFER
 11  3,6  | REBEL
 11  3,6  | REFEL
 11  3,6  | REFL
 11  3,6  | REVIE
 11  4,3  | VEEN
 11  4,3  | VEIN
 11  5,5  | VIBE
 11  1,8  - aBEL
 10  2,8  | BE
 10  4,3  | BEEN
 10  4,3  | BEIN
 10  4,3  | BIEN
 10  4,5  | BNF
 10  6,4  - BOIL
 10  7,3  - CELE
 10  7,3  - CFI
 10  2,8  | EEL
 10  2,6  | EF
 10  1,7  - EF
 10  2,2  | EL
 10  4,3  - ELEC
 10  3,3  | EVER
 10  5,4  - EVOE
 10  3,4  | FERIE
 10  5,4  - FLOE
 10  6,4  - FOIL
 10  4,4  | LENE
 10  5,3  - LICE
 10  4,4  | LINE
 10  4,6  | NEIF
 10  4,6  | NILE
 10  3,6  | REVE
 10  3,6  | RIVE
 10  3,4  | VERI
 10  3,4  | VIRE
 10  1,8  - aLBEE
 10  1,9  - nIEVE
 10  1,9  - nIFE
  9  6,6  | AFB
  9  3,4  | BERE
  9  3,4  | BERI
  9  7,3  - CEIL
  9  7,3  - CLEE
  9  5,4  - EBOE
  9  6,3  - ECB
  9  3,5  - EE
  9  3,7  - EE
  9  2,5  | EEL
  9  2,9  | EEL
  9  4,5  - EL
  9  6,4  - EOLE
  9  5,4  | EVIL
  9  5,4  | FBI
  9  3,4  | FERE
  9  5,5  | FIB
  9  3,4  | FIRE
  9  7,5  - GILE
  9  2,4  | LEE
  9  1,3  - LEEF
  9  1,3  - LEIF
  9  2,2  | LI
  9  2,4  | LIE
  9  1,3  - LIEF
  9  4,4  | LIN
  9  5,5  | LIVE
  9  4,6  | NIL
  9  3,6  | RIBE
  9  3,6  | RIFE
  9  1,5  - TEE
  9  5,4  | VEIL
  9  5,5  | VILE
  9  5,5  | VILI
  9  6,4  - VOE
  9  1,8  - aBLE
  9  1,8  - aLBE
  9  1,8  - aLBI
  9  1,8  - aVIE
  9  1,9  - nEB
  9  1,9  - nEF
  9  1,9  - nIB
  8  6,6  | ABEL
  8  6,6  | ABLE
  8  6,6  | ALBE
  8  6,6  | ALBI
  8  6,6  | ALEF
  8  6,6  | ALIF
  8  6,6  | AVIE
  8  5,5  | BILE
  8  6,4  - BOE
  8  7,3  - CLI
  8  2,6  | EEL
  8  4,4  | ELNE
  8  5,4  | FEIL
  8  5,5  | FIEL
  8  5,5  | FILE
  8  5,5  | FILI
  8  6,4  - FOE
  8  7,5  - GEL
  8  7,5  - GIL
  8  7,5  - GLEE
  8  4,5  | INV
  8  3,3  | LEER
  8  5,4  | LEIF
  8  5,5  | LIEF
  8  4,3  | LIEN
  8  3,3  | LIER
  8  5,5  | LIFE
  8  1,3  - LILE
  8  5,5  | LIV
  8  4,6  | NEIL
  8  7,4  - OLE
  8  3,6  | REEL
  8  3,6  | REV
  8  3,6  | RIEL
  8  3,6  | RIV
  8  3,4  | VER
  8  5,5  | VIL
  8  1,8  - aBIE
  8  1,8  - aFB
  8  1,8  - aIEL
  7  6,6  | ABIE
  7  6,6  | ABL
  7  6,6  | ALB
  7  6,6  | AVE
  7  3,4  | BER
  7  5,4  - BLO
  7  7,3  - CEE
  7  7,3  - CIE
  7  2,9  | EL
  7  1,4  - ELB
  7  1,4  - ELF
  7  1,4  - EVE
  7  3,4  | FER
  7  5,5  | FIL
  7  3,4  | FIR
  7  5,4  - FLO
  7  4,5  | INF
  7  5,6  | IVE
  7  3,4  | LERE
  7  5,5  | LIB
  7  5,5  | LIF
  7  3,4  | LIRE
  7  3,6  | REB
  7  3,6  | REF
  7  3,6  | RIB
  7  3,6  | RILE
  7  5,4  | VEI
  7  5,5  | VIE
  7  5,4  | VII
  7  5,5  | VII
  7  1,9  - nEIF
  6  5,4  - BIO
  6  4,4  | EEN
  6  3,4  | EIRE
  6  2,6  | EL
  6  1,7  - EL
  6  3,5  | ERIE
  6  5,5  | FIE
  6  7,5  - GEE
  6  7,5  - GIE
  6  5,4  - IBO
  6  6,3  - ICE
  6  5,6  | IFE
  6  6,4  - LOE
  6  4,6  | NEE
  6  4,6  | NEI
  6  7,4  - OIL
  6  3,6  | RLE
  6  1,8  - aBL
  6  1,8  - aILE
  6  1,8  - aLB
  6  1,8  - aLEE
  6  1,8  - aVE
  6  1,9  - nILE
  5  6,6  | AB
  5  6,6  | AIEL
  5  6,6  | AILE
  5  6,6  | ALEE
  5  5,5  | BI
  5  6,4  - BO
  5  2,9  | EE
  5  1,4  - EF
  5  3,5  | ERE
  5  5,5  | FI
  5  5,6  | IF
  5  3,5  | IRE
  5  1,3  - LEE
  5  1,3  - LEI
  5  3,4  | LER
  5  1,3  - LIE
  5  3,4  | LIR
  5  3,6  | REL
  5  1,9  - nEIL
  5  1,9  - nIL
  4  6,6  | AIL
  4  6,6  | ALE
  4  2,5  | EE
  4  2,6  | EE
  4  1,7  - EE
  4  2,8  | EE
  4  1,4  - EEL
  4  3,4  | EER
  4  3,4  | EIR
  4  7,5  - GI
  4  5,6  | ILE
  4  5,4  | LEI
  4  5,5  | LIE
  4  1,3  - LL
  4  7,4  - OE
  4  7,4  - OI
  4  3,6  | REE
  4  3,6  | RIE
  4  1,5  - TE
  4  1,8  - aB
  3  6,6  | AL
  3  1,4  - EL
  3  4,5  | EN
  3  3,5  | ER
  3  4,5  | IN
  3  1,3  - LI
  3  5,5  | LI
  3  6,4  - LO
  3  4,6  | NE
  3  3,6  | RE
  3  1,8  - aIL
  3  1,8  - aLE
  3  1,9  - nEE
  3  1,9  - nEI
  2  6,6  | AE
  2  6,6  | AI
  2  1,4  - EE
  2  6,4  - EO
  2  6,4  - IO
  2  1,8  - aL
  1  1,8  - aE
  1  1,8  - aI
  1  1,9  - nE
