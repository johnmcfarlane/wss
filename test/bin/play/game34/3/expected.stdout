 51  3,1  | LEVIER
 42  7,3  - CELEB
 39  3,1  | BELIER
 39  3,1  | EVILER
 39  3,1  | LIEVER
 39  3,6  | RELIVE
 39  3,6  | REVILE
 39  3,1  | VEILER
 36  3,1  | LIEFER
 36  3,6  | REFILE
 36  3,6  | RELIEF
 25  2,8  | BEEF
 24  2,1  | EVIL
 24  3,3  | FEBRILE
 24  2,1  | VEIL
 22  3,4  | VERBILE
 21  4,2  | LEVIN
 21  4,2  | LIVEN
 20  6,6  | ALIVE
 20  3,5  | BRIEF
 20  3,3  | FIBRE
 20  2,2  | FIL
 19  2,8  | BEL
 19  4,3  | EVEN
 19  2,8  | LEV
 19  6,4  - LOVE
 19  4,6  | NEVE
 19  4,6  | NEVI
 19  4,4  | VINE
 19  1,8  - aLIVE
 18  6,6  | ABELE
 18  5,3  | BELIVE
 18  3,5  | BREVE
 18  4,2  | ELFIN
 18  3,5  | ERVIL
 18  4,2  | LEBEN
 18  1,3  - LEVEL
 18  2,4  | LIEF
 18  3,3  | LIVRE
 17  2,8  | BEE
 17  5,3  | BELIEF
 17  6,4  - FOB
 17  7,4  - OBELI
 16  4,4  | BENE
 16  4,4  | BINE
 16  4,4  - FILO
 16  4,4  | FINE
 16  1,3  - LIBEL
 16  6,4  - LOBE
 16  3,4  | VERB
 16  5,3  - VICE
 15  4,4  | BEN
 15  4,4  | BIN
 15  2,9  | ELF
 15  2,9  | EVE
 15  4,2  | FELINE
 15  4,4  | FEN
 15  4,4  | FIN
 15  6,4  - FOIBLE
 15  7,5  - GIVE
 15  6,4  - LOB
 15  4,6  | NEB
 15  4,6  | NIB
 15  3,6  | REIVE
 15  1,8  - aLEF
 15  1,8  - aLIF
 14  5,3  - BICE
 14  3,5  - EEL
 14  3,4  | FERLIE
 14  3,2  | FEVER
 14  5,3  - FICE
 14  3,2  | FIVER
 14  1,3  - LIVE
 14  7,4  - OBE
 14  7,4  - OBI
 14  7,4  - OLIVE
 14  3,3  | VEER
 14  3,3  | VIER
 13  5,3  | BELIE
 13  2,2  | BI
 13  4,3  | BLIN
 13  4,7  - EF
 13  3,2  | ELVER
 13  1,4  - EVIL
 13  3,2  | FIBER
 13  2,4  | FIE
 13  4,3  - FLIC
 13  3,3  | FLIR
 13  7,5  - GIBE
 13  2,3  | IF
 13  1,3  - LEV
 13  4,4  - LEVO
 13  7,4  - OF
 13  5,4  - VIOL
 13  6,4  - VOLE
 12  3,3  | BEER
 12  3,3  | BIER
 12  3,4  | BIRLE
 12  6,4  - BOLE
 12  3,5  | BREE
 12  3,5  | BRIE
 12  7,3  - CLEF
 12  5,7  - FE
 12  3,2  | FLEER
 12  3,2  | FLIER
 12  3,5  | FREE
 12  7,5  - GIB
 12  7,5  - GLEBE
 12  3,2  | LEVER
 12  1,3  - LIEVE
 12  1,3  - LIFE
 12  3,2  | LIVER
 12  4,6  | NIEVE
 12  3,6  | REEF
 12  3,6  | REIF
 12  3,6  | REVEL
 12  3,6  | RIFLE
 12  3,2  | VILER
 12  3,4  | VIRL
 12  6,4  - VOILE
 12  1,8  - aBELE
 12  1,9  - nEVE
 12  1,9  - nEVI
 11  3,4  | BIRL
 11  2,9  | EF
 11  3,2  | FILER
 11  5,5  | FIVE
 11  7,5  - GLIB
 11  2,8  | LEE
 11  2,1  | LEI
 11  2,8  | LEI
 11  1,3  - LIB
 11  3,2  | LIBER
 11  5,5  | LIEVE
 11  3,2  | LIFER
 11  3,6  | REBEL
 11  3,6  | REFEL
 11  4,3  | VEIN
 11  5,5  | VIBE
 10  2,8  | BE
 10  4,3  | BEEN
 10  6,4  - BOIL
 10  2,8  | EEL
 10  2,6  | EF
 10  1,7  - EF
 10  3,3  | EVER
 10  5,4  - FLOE
 10  6,4  - FOIL
 10  5,3  - LICE
 10  4,4  | LINE
 10  4,6  | NEIF
 10  3,6  | RIVE
 10  1,9  - nIEVE
  9  7,3  - CEIL
  9  7,3  - CEL
  9  2,9  | EEL
  9  4,5  - EL
  9  5,4  | EVIL
  9  3,4  | FERE
  9  5,5  | FIB
  9  3,4  | FIRE
  9  2,2  | LI
  9  2,4  | LIE
  9  1,3  - LIEF
  9  4,4  | LIN
  9  5,5  | LIVE
  9  4,6  | NIL
  9  3,6  | RIFE
  9  5,4  | VEIL
  9  5,5  | VILE
  9  6,4  - VOE
  9  1,8  - aBLE
  9  1,9  - nEB
  9  1,9  - nIB
  8  6,6  | ABLE
  8  6,6  | ALEF
  8  6,6  | ALIF
  8  5,5  | BILE
  8  5,5  | FILE
  8  6,4  - FOE
  8  7,5  - GEL
  8  7,5  - GLEE
  8  3,3  | LEER
  8  5,5  | LIEF
  8  4,3  | LIEN
  8  3,3  | LIER
  8  5,5  | LIFE
  8  7,4  - OLE
  8  3,6  | REEL
  8  3,6  | REV
  8  3,6  | RIEL
  7  6,6  | ALB
  7  6,6  | AVE
  7  7,3  - CEE
  7  2,9  | EL
  7  1,4  - ELF
  7  1,4  - EVE
  7  3,4  | FER
  7  5,5  | FIL
  7  3,4  | FIR
  7  5,5  | LIB
  7  3,4  | LIRE
  7  3,6  | REB
  7  3,6  | REF
  7  3,6  | RIB
  7  3,6  | RIF
  7  3,6  | RILE
  7  5,5  | VIE
  7  1,9  - nEIF
  6  5,4  - BIO
  6  2,6  | EL
  6  1,7  - EL
  6  5,5  | FIE
  6  7,5  - GEE
  6  7,5  - GIE
  6  6,3  - ICE
  6  4,6  | NEE
  6  7,4  - OIL
  6  1,8  - aLB
  6  1,8  - aLEE
  6  1,8  - aVE
  5  6,6  | AB
  5  6,6  | ALEE
  5  5,5  | BI
  5  6,4  - BO
  5  1,4  - EF
  5  3,5  | ERE
  5  5,6  | IF
  5  3,5  | IRE
  5  1,3  - LEE
  5  1,3  - LEI
  5  1,3  - LIE
  5  1,9  - nIL
  4  6,6  | AIL
  4  6,6  | ALE
  4  1,4  - EEL
  4  7,5  - GI
  4  5,4  | LEI
  4  5,5  | LIE
  4  7,4  - OE
  4  7,4  - OI
  4  3,6  | REE
  4  1,8  - aB
  3  6,6  | AL
  3  1,4  - EL
  3  4,5  | EN
  3  3,5  | ER
  3  4,5  | IN
  3  1,3  - LI
  3  5,5  | LI
  3  6,4  - LO
  3  4,6  | NE
  3  3,6  | RE
  3  1,8  - aIL
  3  1,8  - aLE
  3  1,9  - nEE
  2  6,6  | AE
  2  6,6  | AI
  2  1,8  - aL
  1  1,8  - aE
  1  1,8  - aI
  1  1,9  - nE
